easybuild/easyconfigs/e/expat/expat-2.2.10.eb

```diff
@@ -0,0 +1,18 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'expat'
+version = "2.2.10"
+
+homepage = 'http://expat.sourceforge.net/'
+description = """Expat is an XML parser library written in C. It is a stream-oriented parser in which an application
+ registers handlers for things the parser might find in the XML document (like start tags)"""
+
+toolchain = SYSTEM
+toolchainopts = {'pic': True}
+
+# http://downloads.sourceforge.net/project/expat/expat/2.2.10/expat-2.2.10.tar.xz
```

Comment:
delete line

Reply:
I have added the line for convenience, to have the direct download link for quick manual checks in case of issues